pkg/cmd/controller/root.go

```diff
@@ -18,8 +18,15 @@ func NewCommand() *cobra.Command {
 		Short: "OCI image build controller using buildkit",
 	}
 	cmd.PersistentFlags().StringVarP(&cfgFile, "config", "c", "hephaestus.yaml", "configuration file")
+
+	runGCCmd := newRunGCCommand()
+	runGCCmd.Flags().Bool("enabled", true, "Enable the auto image builder clean up.")
+	runGCCmd.Flags().Int("maxIBRetention", 5, "Maximum number of image builds to retain."+
```

Comment:
Just in the interest of being consistent, the flags should be defined in the `newXYZCommand()` method like the others?